modules/cql/test/blaze/elm/compiler/type_operators_test.clj

```diff
@@ -571,6 +571,78 @@
 ;; String
 ;;
 ;; If the argument is null, the result is null.
+(deftest compile-converts-to-string-test
+  (testing "Boolean"
+    (are [x] (true? (core/-eval (tu/compile-unop elm/converts-to-string elm/boolean x)
+                                    {} nil nil))
+      "true"
+      "false"))
+
+  (testing "Integer"
+    (are [x] (true? (core/-eval (tu/compile-unop elm/converts-to-string elm/integer x)
+                                    {} nil nil))
+      "-1"
+      "0"
+      "1"))
+
+  (testing "Decimal"
+    (are [x] (true? (core/-eval (tu/compile-unop elm/converts-to-string elm/decimal x)
```

Comment:
I would only use one test value for each type, because the result only depends on the type and not on the actual value.